find.go

```diff
@@ -129,24 +129,17 @@ func (f *finder) structType(name *ast.Ident, t *ast.StructType) {
 	}
 }
 
-const _required = "// required"
+var _requiredRegex = regexp.MustCompile(`^(\/\/\s*required)(.*)`)
 
 func isRequiredComment(c *ast.Comment) bool {
-	if c.Text == _required {
-		return true
-	}
-
-	if !strings.HasPrefix(c.Text, _required) {
+	matches := _requiredRegex.FindStringSubmatch(c.Text)
+	if len(matches) == 0 {
 		return false
 	}
-
-	for _, r := range c.Text[len(_required):] {
+	sepAndDescription := matches[2]
+	for _, r := range sepAndDescription {
 		return !unicode.IsLetter(r) && !unicode.IsDigit(r) && r != '_'
 	}
-
-	// Impossible:
-	// If the comment is not "// required", but it starts with that,
-	// the loop above will always return before we get here.
-	// This is just to make the compiler happy.
-	return false
+	// Nothing after the `// required` comment.
```

Comment:
nit: not sure I follow this comment. the code still reaches here if you have `// required: some context` right?

Reply:
No, if there's anything after `// required`, then `sepAndDescription` is non-empty, in which case the above `for` loop runs exactly once. My understanding is that it's a `for` loop as a simple way of just grabbing the first rune and checking whether it's alphanumeric or not.